### ISSUE TITLE
Using usbipd to connect to a USB device in ubuntu, I can't find the device node under "/dev/" and no mount path is found

### ISSUE THREAD

**Issue by xfan-hack** [microsoft/WSL#12215]
### Windows Version

10.0.26100.2161

### WSL Version

2.3..24.0

### Are you using WSL 1 or WSL 2?

- [x] WSL 2
- [ ] WSL 1

### Kernel Version

Linux version 5.15.153.1-microsoft-standard-WSL2 (root@941d701f84f1) (gcc (GCC) 11.2.0, GNU ld (GNU Binutils) 2.37) #1 SMP Fri Mar 29 23:14:13 UTC 2024

### Distro Version

Ubuntu 22.04.5

### Repro Steps

Execute commands under powershell: usbipd.exe list, usbipd.exe bind --busid 5-3, usbipd.exe attach --wsl --busid 5-3
 In ubuntu system, execute lsusb, you can see the USB device, but in the /dev directory, there is no But in the /dev directory, there is no corresponding device name, and I can't find any information about the mount path using mount -l.

### Expected Behavior

I'd like to be able to see the mount paths etc. of USB devices on ubuntu systems in wsl

### Actual Behavior

Provide a solution or documentation that addresses the above issues

**Comment by zcobol** [comment]
@xfan-hack what is your expectation of `usbipd`? If you try to mount a storage device use `drvfs` option instead. `usbip` is for attaching other kind of USB devices, i.e. cameras, etc.

On WSL side add a mount point under `/mnt` then mount your flash drive, Windows `F:` in this example:

`zcobol@texas:~$ sudo mount -t drvfs F: /mnt/f `

then list its contents:

```
zcobol@texas:~$ tree -L 1 /mnt/f
/mnt/f
├── autorun.ico
├── autorun.inf
├── boot
├── casper
├── EFI
├── efi.img
├── isolinux
├── md5sum.txt
├── syslinux.cfg
└── System Volume Information

6 directories, 5 files
```

The USB driver has latest KDE NEON on it!

**Comment by xfan-hack** [comment]
> # Logs are required for review from WSL team
> If this a feature request, please reply with '/feature'. If this is a question, reply with '/question'. **Otherwise please attach logs by following the instructions below**, your issue will not be reviewed unless they are added. These logs will help us understand what is going on in your machine.
> 
> How to collect WSL logs
> # View similar issues
> Please view the issues below to see if they solve your problem, and if the issue describes your problem please consider closing this one and thumbs upping the other issue to help us prioritize it!
> 
> ### Open similar issues:
> * [Unable to mount a usb mass storage device (#10446)](https://github.com/microsoft/WSL/issues/10446),  similarity score: 0.78
> * [Accessing USB storage devices in WSL 2? (#7770)](https://github.com/microsoft/WSL/issues/7770),  similarity score: 0.74
> 
> ### Closed similar issues:
> * [Can't Attach USB Devices to WSL (#9990)](https://github.com/microsoft/WSL/issues/9990),  similarity score: 0.78
> * [Can't see USB in `lsblk` (#8541)](https://github.com/microsoft/WSL/issues/8541),  similarity score: 0.76
> * [usbipd: error: WSL 'usbip' client not correctly installed. (#8192)](https://github.com/microsoft/WSL/issues/8192),  similarity score: 0.75
> 
> > Note: You can give me feedback by thumbs upping or thumbs downing this comment.
Yes, my problem, and "Open similar issues:(#10446) (#7770)" is similar, the essence of these problems is that, in ubuntu system in wsl, although I can see the inserted USB device through lsusb, I can't see the device under /dev/ directory, and I don't see the path of the mounted USB device. directory, and I don't see the path where the USB device is mounted. In short, it is that ubuntu system in wsl can't access the USB and other mobile devices.
*** Translated with www.DeepL.com/Translator (free version) ***

**Comment by marksteward** [comment]
Non-partitioned drives aren't accessible using drvfs, and wsl --mount is unpredictable, doesn't work on many drives, and has crashed my system multiple times. Just add some mass storage drivers to the kernel - see also #11335.

**Comment by bugparty** [comment]
Don't lose hope, I wrote an tutorial and it is not complex, just take 10 minutes without any sideeffects.
see here: https://www.linkedin.com/pulse/wsl2-sd-card-support-fix-guide-bowen-han-34x3c/
